### DISCUSSION TITLE
Winpilot 3.4.0 (Stargate) ⊙🛸🚀

### DISCUSSION THREAD

**Discussion by Belim** [builtbybel/Bloatynosy#171] [Announcements]
### Important note: Bloatynosy/AI has now evolved into Winpilot. Check out [3.0.0](https://github.com/builtbybel/Winpilot/releases/tag/3.0.0) for more details 

<blockquote class="twitter-tweet"><p lang="en" dir="ltr">My Winpilot is supported by Clippy 🛩️😂📎<a href="https://t.co/PkRbJxXfVg">https://t.co/PkRbJxXfVg</a> <a href="https://t.co/whQL0LHulG">pic.twitter.com/whQL0LHulG</a></p>&mdash; Belim (@builtbybel) <a href="https://twitter.com/builtbybel/status/1772353175028187507?ref_src=twsrc%5Etfw">March 25, 2024</a></blockquote>

# Changelog
### The recently introduced new, simpler "Stargate" UI (Codename for release) for Winpilot is now making its way into Winpilot. It's designed with a classic feel, structured like an assistant akin to the old Bloatynosy app. The "AI" stuff takes a backseat here, focusing on the core task: debloating and customizing the system in 3-4 steps. Clippy now enjoys a bit more prominence, and you'll always have him by your side.

**Under-the-hood changes:**
- Optimized and generally faster startup
- Optimized high DPI scaling (for handhelds like Legion GO, Rog Ally) You can zoom in and out within the app by holding Ctrl and scrolling the mouse wheel
- Migration of some older Bloatynosy functions and features

![Screenshot 2024-04-10 083714](https://github.com/builtbybel/Winpilot/assets/57478606/c7dac870-c7bf-4858-8d83-eb4e713c6d5a)

<hr /><em>This discussion was created from the release <a href='https://github.com/builtbybel/Winpilot/releases/tag/3.4.0'>Winpilot 3.4.0 (Stargate) ⊙🛸🚀</a>.</em>

**Comment by Hawkeye4040** [comment]
It's about time you decided to use grown up words for your software.

**Reply by Belim** [reply]
ive upgraded from junior to senior dev status 🥲

**Comment by ebug64** [comment]
Can't download due to virus warning on Defender, Windows 11.

**Reply by Belim** [reply]
Were already reported! What protection definition do you have?

**Comment by MagicSergio91** [comment]
Hello, 
I have a prob with .net framework... I can't be able to remove .net framework 4.8 to install V4.0 ... Ca  you help me ... thanks ...
![WinPilot1](https://github.com/builtbybel/Winpilot/assets/155170239/dca7cd32-554e-4fbf-ba4d-f79585142378)
![WinPilot2](https://github.com/builtbybel/Winpilot/assets/155170239/ed03b73b-3d31-4ad1-a34b-2681147a0c1a)
